.github/CODEOWNERS

```diff
@@ -131,7 +131,7 @@
 /scripts/typescript_checks*.py @ankita240796 @DubeySandeep
 /scripts/linters/ @DubeySandeep @Hudda
 /scripts/linters/excluded_any_type_files.json @nishantwrp
-/ubuntu_dockerfile @ezl-13 @kevinlee12
+/ubuntu_dockerfile @ezl-13 @DubeySandeep
```

Comment:
@kevinlee12, I don't see any user with username @ezl-13, is that an expected user? if now should we remove him?

Reply:
yeah, he was the author of this script. I don't think he's active any more, so I removed him for now.